docs/docs/docs/getting-started/installation.md

```
@@ -341,6 +341,28 @@ You can run the app after closing the terminal or restating the vscode using the
 
 **Note:** These commands will start the server in development mode.
 
+## Sample Users
+
+Below is a table of sample user data. 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)